tests/integration/targets/aws_secret/tasks/main.yaml

```diff
@@ -12,3 +12,5 @@
   - include_tasks: 'basic.yml'
   # Permissions missing
   #- include_tasks: 'rotation.yml'
+  # Permissions missing and botocore>=1.23.21
```

Comment:
So these can be enabled as it is possible to pull in custom versions in the tests for example:
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/dynamodb_table/meta/main.yml

then you just provide the custom virtual_env python with newer boto like this:
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/dynamodb_table/tasks/main.yml#L591-L592